migrations/2019_09_28_222800_create_users_table_preferences_columns.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('users', function (Blueprint $table) {
```

Comment:
Why don't we have a dedicated table for user preferences like with notification preferences? I'm not sure how I feel about extensions modifying the user table this much, it might lead to unexpected conflicts.